src/standard/annotations.html

```diff
@@ -236,7 +236,7 @@
 
     // push configuration references at configure time
     _configureAnnotationReferences: function() {
-      var notes = this._notes;
+      var notes = this._notes || [];
```

Comment:
Why is this changed needed?

Reply:
Good question.  When `_prepTemplate` throws under a `strictTemplatePolicy` violation, it leaves `_notes` uninitialized, which was previously not a case that could ever occur since `_prepAnnotations` comes after that and unconditionally sets `_notes`.  So this just provides a good fallback behavior under the violation case.